chainerx_cc/chainerx/native/CMakeLists.txt

```
@@ -59,6 +60,14 @@ if(${BLAS_FOUND})
     endif()
 endif()
 
+# LAPACK
+if(${LAPACK_FOUND})
+MESSAGE( STATUS "LAPACK is found, defining CHAINERX_ENABLE_LAPACK variable.")
+add_definitions(-DCHAINERX_ENABLE_LAPACK=${LAPACK_FOUND})
```

A detail again but I think this is easier to comprehend.
```suggestion
add_definitions(-DCHAINERX_ENABLE_LAPACK=1)
```